Resources/Prototypes/Entities/Objects/Consumable/Food/meals.yml

```diff
@@ -216,7 +216,7 @@
 # Tastes like sweet potato.
 
 - type: entity
-  name: Cuban carp
+  name: cuban carp
```

Comment:
Cuba is a place, and therefore a proper noun. Proper nouns should remain capitalized.